src/Profile/defaults.js

```diff
@@ -0,0 +1,18 @@
+import { getLocale, getLanguage, getCountryCode, getLatLon } from './helpers'
+
+export const defaultProfile = {
+  ageRating: 'adult',
+  city: 'New York',
+  countryCode: getCountryCode('US'),
+  ip: '127.0.0.1',
+  household: 'b2244e9d4c04826ccd5a7b2c2a50e7d4',
+  language: getLanguage('en'),
+  latlon: getLatLon([40.7128, 74.006]),
+  locale: getLocale('en-US'),
+  mac: '00:00:00:00:00:00',
+  operator: 'Metrological',
+  platform: 'Metrological',
+  packages: [],
+  uid: 'ee6723b8-7ab3-462c-8d93-dbf61227998e',
+  stbType: 'Metrological',
+}
```

Comment:
@wouterlucas @erikhaandrikman @woutermeek any opinion about these default values? Are they the best "sensible defaults"?

Reply:
I'd prefer San Francisco over New York (just kidding). No they look good.

Generic question though, will the meaning of these values be documented somewhere? Will we rely on a fixed set of values?

To elaborate, the "ageRating" triggered me. I earlier had an `integer` for that, or number, to define age. Now the default is set to `adult`, what other values can a developer expect? We'll need to document what these values mean and what can a developer expect for each one of them.

Granted, we may pull that apart of this PR and do the doc update separately. To avoid making the docs a long pole and block this PR.

Reply:
I actually think we should make a habit out of adding documentation as part of each feature PR (if there are changes that need documentation).

In this case we should probably document the default values. But also how they can be customized (during development) via settings.json.

Reply:
I have updated the documentation with the default values added.